### PR TITLE
Enable debug logging add recommendation

### PR DESCRIPTION
**Pull request by YunchuWang**
## Description

Fixes #

---

### PR information

- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azure_functions_worker/logging.py [hunk 1]
```diff
@@ -76,6 +76,12 @@ def enable_console_logging() -> None:
         logger.addHandler(handler)
 
 
+def enable_debug_logging_recommendation():
+    logging.info("To enable debug level logging, please refer to "
+                 "https://docs.microsoft.com/en-us/azure/azure-functions/"
+                 "functions-app-settings#python_enable_debug_logging")
```
Comment (vrdmr) [inline azure_functions_worker/logging.py:82]: nit; Can you make this into an aka.ms link? Work with @shreyabatra4 to create a new link

**Review by vrdmr — COMMENTED** [review]
nit changes requested.

You missed another comment from @gavin-aguiar - https://github.com/Azure/azure-functions-python-worker/pull/943#pullrequestreview-856093164. Could you please address that in here?

File: azure_functions_worker/logging.py (hunk 1 above)
Comment (shreyabatra4) [reply azure_functions_worker/logging.py:82]: here it is: aka.ms/python-enable-debug-logging

File: azure_functions_worker/dispatcher.py [hunk 2]
```diff
@@ -459,6 +461,7 @@ async def _handle__function_environment_reload_request(self, req):
         try:
             logger.info('Received FunctionEnvironmentReloadRequest, '
                         'request ID: %s', self.request_id)
+            enable_debug_logging_recommendation()
```
Comment (gavin-aguiar) [inline azure_functions_worker/dispatcher.py:464]: @vrdmr Should we pass this along with the above 'Received FunctionEnvironmentReloadRequest' log, so we have only one grpc call here? Same with the worker init request

Comment (vrdmr) [reply azure_functions_worker/dispatcher.py:464]: During the init, it won't matter much as grpc overhead for only one call is very low. It is when that call is repeated multiple times, then it'll start hampering us.

Also, as this is something that we are expecting customers to observe and take action if necessary, I think we shouldn't mix it with the other logs.

I would recommend keeping as it is.

File: azure_functions_worker/dispatcher.py [hunk 3]
```diff
@@ -24,20 +24,21 @@
...
 from .constants import (PYTHON_THREADPOOL_THREAD_COUNT,
                         PYTHON_THREADPOOL_THREAD_COUNT_DEFAULT,
                         PYTHON_THREADPOOL_THREAD_COUNT_MAX_37,
                         PYTHON_THREADPOOL_THREAD_COUNT_MIN,
                         PYTHON_ENABLE_DEBUG_LOGGING)
+from .extension import ExtensionManager
 from .logging import disable_console_logging, enable_console_logging
+from .logging import enable_debug_logging_recommendation
 from .logging import (logger, error_logger, is_system_log_category,
                       CONSOLE_LOG_PREFIX)
```
Comment (vrdmr) [inline azure_functions_worker/dispatcher.py:37]: Why don't we merge these three imports into one?

**Review by vrdmr — APPROVED** [review]
LGTM. Thanks for adding this log.

nit - please make the one minor change in the review..

**Pull request merged**